Cargo.toml

```diff
@@ -41,6 +42,7 @@ tracing = { version = "0.1.40" }
 tracing-appender = { version = "0.2.3" }
 tracing-attributes = "0.1.27"
 tracing-subscriber = { version = "0.3.18", default-features = true, features = ["env-filter", "json", "registry"] }
+console-subscriber = { version = "0.4.0", optional = true }
```

Comment:
Does it help if this is in release too or its only for local debugging?

Reply:
It's only for local debugging, on to of that it only works if we enable these flags:

```
RUSTFLAGS="--cfg tokio_unstable"
```